Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
```

Comment:
No there are URLs for specifics versions like `https://files.magerun.net/n98-magerun-1.93.0.phar`    
The URL `https://files.magerun.net/n98-magerun.phar` is like `https://files.magerun.net/n98-magerun-latest.phar` it is the latest release.    
Do I have to remove version line and specify `https://files.magerun.net/n98-magerun-latest.phar` to make it more simple ?